src/Service/ActivityTransformer.php

```diff
@@ -110,39 +103,68 @@ private function resetTechTypeCollections(Activity $entity): void
      * @return Activity
      * @throws EntityNotFound
      */
-    public function transform(
+    public function createTransform(
         ActivityDTO $dto
     ): Activity {
 
-        if ($dto->id !== null) {
-            $entity = $this->activityRepo->find($dto->id);
-            if (!$entity) {
-                $entityNotFound = new EntityNotFound(
-                    Activity::class,
-                    $dto->id,
-                    'No activity found.'
-                );
-                throw $entityNotFound;
-            }
-            $this->resetTechTypeCollections($entity);
-        } else {
-            $entity = new Activity();
-        }
+        $entity = new Activity();
         $entity->setName($dto->name);
         $entity->setDescription($dto->description);
         $entity->setApplicationDeadline($dto->applicationDeadline);
         $entity->setFinalDeadline($dto->finalDeadline);
-        $entity->setCreatedAt($dto->createdAt);
-        $entity->setUpdatedAt($dto->updatedAt);
         $entity->setStatus($dto->status);
 
         /** @var User $tempUser */
         $tempUser = $this->userRepo->find($dto->owner);
```

Comment:
The current user should be passed from the controller as the owner. With this change I could create a job with someone else as owner.